apollo-ios-pagination/Sources/ApolloPagination/GraphQLQueryPagerOutput.swift

```diff
@@ -14,7 +14,7 @@ public struct PaginationOutput<InitialQuery: GraphQLQuery, PaginatedQuery: Graph
   public let nextPages: [PaginatedQuery.Data]
 
   /// The source of the most recent `Output`: either from the cache or server.
-  public let updateSource: UpdateSource
```

Comment:
Since it's become a bit awkward having this in two places, I'm wondering if we can just remove it from `PaginationOutput` completely? The only place it looks like it's being used right now is a few unit tests, and to pass the `updateSource` through to the `Output` result.

Could we just add the `updateSource` to the result value of the `GraphQLQueryPagerCoordinator` and then pass it through that way instead of having it as a stored property at all?

Reply:
Totally possible.

Reply:
@AnthonyMDev done in https://github.com/apollographql/apollo-ios-dev/pull/252/commits/523255f01c7f00f4485dde6b8768297dfccd9378